frontend/src/core/codemirror/completion/completer.ts

```diff
@@ -47,9 +47,6 @@ export async function completer(
       documentation: tooltip.html ?? null,
     });
   }
-  if (tooltip) {
-    return null;
-  }
```

Comment:
another thing @akshayka , this removal introduces some side effects. It's not ideal.
```
df = pl.DataFrame({
  "a": [1,2],# <--- cursor here will pop up an autocomplete for Dataframe which overrides Enter key
})
```

I am keen to revert this, an ideal solution is to display as just a tooltip, it would take me too long to implement imo :/